Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -68,7 +68,7 @@ public void MutateSeed(SeedData seed, float severity)
         MutateBool(ref seed.Sentient       , true  , 10 , totalbits , severity);
         MutateBool(ref seed.Ligneous       , true  , 10 , totalbits , severity);
         MutateBool(ref seed.Bioluminescent , true  , 10 , totalbits , severity);
-        MutateBool(ref seed.TurnIntoKudzu  , true  , 10 , totalbits , severity);
```

Comment:
In the future, avoid making unrelated balance changes in the same PR as content changes. It does happen that I currently think kudzu happens too often, so let's keep this here for now.

Reply:
Alright, I'll keep that in mind, I thought it'd be fine since it was part of the same mechanic.

Reply:
Oh, I see now, you're saying that turning into kudzu is just like turning into a separate plant, so they should be balanced together. Personally I think turning into kudzu is much worse :p

Reply:
It definitely is much worse, but I really saw no reason why it should have an even higher chance than turning into an actual useful plant (when that is possible in the first place), not to mention that like you said kudzu pops up too often, especially considering how bad it can get